# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1426692630 14400
#      Wed Mar 18 11:30:30 2015 -0400
# Node ID 656cb18bf858d35d73d28c10a24a647891b508b2
# Parent  e78c0514523d3af17634d36f13a0551da4359bc4
finish job of restoring strmatch (bug #44562)

* scripts/deprecated/module.mk (deprecated_FCN_FILES): Remove
strmatch.m from the list.
* NEWS: Remove strmatch from the list of functions deprecated for
version 4.0.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -251,17 +251,16 @@ Summary of important user-visible change
       fnmatch              | glob or regexp
       loadaudio            | audioread
       luinc                | ilu or ichol
       mouse_wheel_zoom     | mousewheelzoom axes property
       nfields              | numfields
       octave_tmp_file_name | tempname
       playaudio            | audioplayer
       saveaudio            | audiowrite
-      strmatch             | strncmp or strcmp or regexp
       syl                  | sylvester
       usage                | print_usage
 
       allow_noninteger_range_as_index
       do_braindead_shortcircuit_evaluation
       setaudio
 
  ** The following functions were deprecated in Octave 3.8 and will be
diff --git a/scripts/deprecated/module.mk b/scripts/deprecated/module.mk
--- a/scripts/deprecated/module.mk
+++ b/scripts/deprecated/module.mk
@@ -26,17 +26,16 @@ deprecated_FCN_FILES = \
   deprecated/nfields.m \
   deprecated/octave_tmp_file_name.m \
   deprecated/playaudio.m \
   deprecated/re_read_readline_init_file.m \
   deprecated/read_readline_init_file.m \
   deprecated/saveaudio.m \
   deprecated/saving_history.m \
   deprecated/setaudio.m \
-  deprecated/strmatch.m \
   deprecated/syl.m \
   deprecated/usage.m
 
 FCN_FILES += $(deprecated_FCN_FILES)
 
 PKG_ADD_FILES += deprecated/PKG_ADD
 
 DIRSTAMP_FILES += deprecated/$(octave_dirstamp)
